### PR TITLE
Added looping inside an Object

### PR DESCRIPTION
**Pull request by JFolberth**
Added looping inside an object to help support advanced usage. Intention is to illustrate how this can be used for templating at some point.

Ran this on a local ADO agent and confirm it works as expected.

### CODE REVIEW

**Comment by jborsecnik** [comment]
@juliakm, 

Can you review the proposed changes? 

IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by juliakm** [comment]
@JFolberth This is a great addition! I've made a couple of small edits and will merge.

**Comment by juliakm** [comment]
#sign-off

File: docs/pipelines/process/templates.md [hunk 1]
```diff
@@ -97,6 +97,22 @@ steps:
 - ${{ each value in parameters.listOfStrings }}:
   - script: echo ${{ value }}
 ``` 
+Additionally you can iterate through nested elements within an object
```
Comment (Jak-MS) [inline docs/pipelines/process/templates.md:100]:
```suggestion

Additionally, you can iterate through nested elements within an object.

```

**Pull request merged**